scripts/2_initiate_token.sh

```diff
@@ -18,10 +18,12 @@ curl -s -X 'POST' \
 }'
 )
 
-echo $heidi_response | jq -r
-
 echo "Loaded into local database"
 
+sleep 0.2
```

Comment:
I'm not sure why this would help as I can't see what the race condition can be between. The curl call shouldn't return until everything has completed on the backend. Are we sure the curl is succeeding?